source/threads.tex

```diff
@@ -851,10 +851,10 @@
     // \ref{stopcallback.cons}, constructors and destructor
     template<class C>
     explicit stop_callback(const stop_token& st, C&& cb)
-        noexcept(is_nothrow_constructible_v<Callback, C>);
+      noexcept(is_nothrow_constructible_v<Callback, C>);
     template<class C>
     explicit stop_callback(stop_token&& st, C&& cb)
```

Comment:
I think we want to indent this as opposed to de-indenting the next line.

Reply:
Fixed, thanks.